user.env

```diff
@@ -2,3 +2,5 @@ local_state=./local_state
 
 TIME_ZONE=Europe/London
 
+# Set a gnome-keyring password
+# KEYPASS=password
```

Comment:
One small comment: is this specific to the Minecraft container? If so, maybe it should go into `env/minecraft.env`. It doesn't exist yet, but `run-gow` will automatically pick it up if you create one.

Reply:
I'm on vacation right now, but can rebase next week. It currently is only on this container, however, if gnome-keyring is reused on other containers it would need to be able to use the same password.

Reply:
Makes sense. Thanks!